pages/[league]/player/[id].tsx

```diff
@@ -46,6 +50,33 @@ const fetchPlayerName = (league: League, playerId: string) =>
   );
 
 export default ({ playerId, league }: { playerId: string; league: League }) => {
+  const router = useRouter();
+
+  const { portalView } = router.query;
+
+  const isIndexView = useMemo(
+    () => Boolean(portalView === undefined),
+    [portalView],
+  );
```

Comment:
A few comments in one here:
1. I'd suggest against using `Boolean()` here to cast this value when you can easily do so with a simple check to see if it is falsy (i.e. `!variable` instead of `variable === undefined`).
2. The use of useMemo seems like a premature optimization. This isn't a value we are recalculating much so we should just be able to store it on its own.
3. Let's rename this variable to a name that better describes what it *does* rather than what it *is*.

```suggestion
  const shouldShowIndexView = !portalView;
```